source/js/components/petition/petition.jsx

```diff
@@ -170,6 +170,10 @@ export default class Petition extends React.Component {
         return reject(new Error(`missing required comment`));
       }
 
+      if (comment && comment.length >= 500) {
```

Comment:
@alanmoo from your comment

> Petition comments are limited to 500 characters in Salesforce.

Is 499 or 500 the max characters allowed??? If it's 500 this should be changed to `>`.

Reply:
Since `500` is being used multiple times in this file, can we turn `500` into a variable with inline comment to describe why 500?

Reply:
honestly, I don't want to take that risk. If Salesforce says it's 500, I'd rather stick with 499 as limit.

Reply:
That said, using constants makes sense, but we might need more than one constant, even if the value is the same across multiple constants.